nle/scripts/ttyplay.py

```diff
@@ -165,10 +167,14 @@ def main():
     FLAGS = parser.parse_args()
 
     if FLAGS.filename == "-":
-        fd = os.dup(0)
+        f = os.fdopen(os.dup(0), "rb")
```

Comment:
just curious as two what youre doing here?  is this is a command line interface that allows you to read from std in? if so, why are you mapping std out to std in below? appreciate its not a real change, just wondering.

Reply:
That's a good question. This is some POSIX magic here: Doing it this way allows us to run this via

```shell
python nle/scripts/ttyplay.py - < my-recording.ttyrec
```

and still do the `ioctl` on fd 0 and *use the keyboard to jump around*. Those keyboard actions are also read from fd 0. I am honestly not quite sure why this works. I think I took the idea from here: https://github.com/ovh/ovh-ttyrec/blob/master/ttyplay.c#L306